packages/react-ui-components/src/stories/foundation/Introduction/Introduction.stories.mdx

```diff
@@ -185,3 +185,55 @@ import { initialize, StyleMode } from '@kelvininc/[react|angular]-ui-components'
 initialize({ baseAssetsUrl: '/clients/' });
 
 ```
+
+## Aditional Information
+
+When using components with props that can assume the value undefined, if no default value is defined, Stencil will consider the undefined value after the first render.
+
+Considering the following example where, tooltipDelay prop can be undefined:
+
+```tsx
+const ComponentWrapper({
+	text,
+	tooltipText,
+	tooltipDelay,
+	onButtonClick
+}) {
+	return (
+		<KvTooltip text={tooltipText} delay={tooltipDelay}>
+			<KvActionButtonText
+				type={EActionButtonType.Primary}
+				text={text}
+				size={EComponentSize.Small}
+				onClickButton={this.onButtonClick}>
+			</KvActionButtonText>
+		</KvTooltip>
+	);
+}
+```
+
+At the first time that the button is hovered, the tooltip will assume a 1 second delay that is defined in our KvTooltip component. However, if you pass the prop to the tooltip and its value is undefined, Stencil will consider the prop as undefined.
+
+If you want your undefined props to assume a default value, please consider defining it like it`s shown in the example bellow:
```

Comment:
Suggestion

```suggestion
At first render, the `KvTooltip` will assume its prop default value of 1 second. However, when the `ComponentWrapper` re-renders, this prop value will be set to `undefined` overwriting the previous value of 1 second. This happens because the `KvTooltip` doesn't fall back to the default value when a prop is `undefined`, it simply uses that value when the component is mounted.

If you want to work around this behaviour please consider setting your drilled prop to a default value as it's shown in the example below:
```

Reply:
Excellent improve! Will update 👍